### DISCUSSION TITLE
Adding the option to enable versioning

### DISCUSSION THREAD

**Discussion by informatJonas** [fourdotsix/tenant-buckets#14] [Ideas]
Hello,

is there a possibility that you provide the option "versioning" for bucket creation?

This would require the following script to be executed after the bucket is created:
```php
$client->putBucketVersioning(
	[
		'Bucket'                  => 'bucket-name',
		'VersioningConfiguration' => [
			'Status' => 'Enabled',
		],
	]
);
```

**Comment by secrethash** [comment]
Hey, can you create a pull request for this? I will verify and merge it.

Thanks.
